File: Nível 2/1035 - Teste de Seleção 1/main.lua
valores = io.read()

nmA, nmB, nmC, nmD = valores:match("(%S+)%s+(%S+)%s+(%S+)%s+(%S+)")

numeroA = tonumber(nmA)
numeroB = tonumber(nmB)
numeroC = tonumber(nmC)
numeroD = tonumber(nmD)

if numeroB > numeroC and numeroD > numeroA and (numeroC + numeroD) > (numeroA + numeroB) and numeroC > 0 and numeroD > 0 and numeroA % 2 == 0 then
    print("Valores aceitos")
else
    print("Valores nao aceitos")
end

File: Nível 2/1035 - Teste de Seleção 1/main.py
valores = input("Digite os valores")

nmA,nmB,nmC,nmD = valores.split()

numeroA = int(nmA)
numeroB = int(nmB)
numeroC = int(nmC)
numeroD = int(nmD)

if numeroB > numeroC and numeroD > numeroA and (numeroC + numeroD) > (numeroA + numeroB) and numeroC > 0 and numeroD > 0 and numeroA % 2 == 0:
    print("Valores aceitos")
else:
    print("Valores nao aceitos")